### ISSUE TITLE
[Bug] Folder names aren't wrapped in quotes in adb shell commands, leading to errors

### ISSUE THREAD

**Issue by vesper8** [VRPirates/rookie#112]
Hey! Thanks a lot for your work on this. I've been trying to use 2.29.2 all day today and it succeeds for a lot of games but constantly fails for many others. I had a closer look at the debug log and it's quite clear what's going on. As the title says, the game folder is being passed without quotes when executing adb shell commands which leads to a certain error.

Here's a few examples:

```

10:38:03.532 PM (UTC): [INFO] [ADB.RunAdbCommandToString] Running command:  -s 1WMHHA41M71511 shell df
10:38:08.365 PM (UTC): [INFO] [ADB.RunAdbCommandToString] Running command:  -s 1WMHHA41M71511 shell rm -rf /sdcard/Android/obb/Redacted v75+1.0.021 -VRP && mkdir /sdcard/Android/obb/Redacted v75+1.0.021 -VRP
10:38:08.412 PM (UTC): [ERROR] [ADB.RunAdbCommandToString] rm: Unknown option 'VRP' (see "rm --help")
```

```
10:48:04.939 PM (UTC): [INFO] [ADB.RunAdbCommandToString] Running command:  -s 1WMHHA41M71511 shell rm -rf /sdcard/Android/obb/Redacted v75+1.0.021 VRP && mkdir /sdcard/Android/obb/Redacted v75+1.0.021 VRP
10:48:04.987 PM (UTC): [ERROR] [ADB.RunAdbCommandToString] mkdir: 'Redacted': Read-only file system
mkdir: 'v75+1.0.021': Read-only file system
mkdir: 'VRP': Read-only file system
```

I also ran the command `adb shell ls /sdcard/Android/obb`

And can also see that these problem games all have created folders in `/sdcard/Android/obb` that are not in the `com.name.app` format so no wonder it's failing.

I hope the fix is simple. Many thanks again!

**Comment by Fenopy** [comment]
I appreciate your quality issue report.

**Comment by Fenopy** [comment]
This is now fixed in 497c670a989b117b0d228a080638199d2cd4eae7.

**Comment by vesper8** [comment]
> This is now fixed in [497c670](https://github.com/VRPirates/rookie/commit/497c670a989b117b0d228a080638199d2cd4eae7).

Awesome thank you!!

Is there a way to get the beta build short of building it myself? Like is there a nightly build somewhere or.. I must either wait or build it myself?

Cheers!
